apps/yapms/src/routes/app/imported/+page.svelte

```diff
@@ -0,0 +1,137 @@
+<script lang="ts">
```

Comment:
Much of this page is a duplicate of the other page. We should move the layout out a common route.

Reply:
Putting this in a separate PR